EpicLoot/EpicLoot.cs

```diff
@@ -179,6 +181,7 @@ public class EpicLoot : BaseUnityPlugin
         private float _worldLuckFactor;
 
         [UsedImplicitly]
+        [HarmonyPriority(Priority.VeryLow)]
```

Comment:
This does nothing, as this is a unity call, not a harmony patch.

Reply:
I did test, it _seems_ to change the order that the mod runs the Awake call.  EpicLoot loaded last now in my Unity log.

Reply:
Removed.